src/app/shared/services/http.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { SearchResponse } from '../models/search-response.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HttpService {
+
```

Comment:
it will be better to name this service in a more meaningful way, the name of a service should tell us exact what is is made for

Reply:
we could implement a lot of services which will be able to comunicate with API via HTTP